### PR TITLE
:es6: onSlotLoad

### PR DESCRIPTION
**Pull request by regiskuckaertz**
With some type definitions for dealing with GPT.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/types.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// @flow
+export type SingleSizeArray = Array<number>;
+export type NamedSize = string;
+export type SingleSize = SingleSizeArray | NamedSize;
+export type MultiSize = Array<SingleSize>;
+export type GeneralSize = SingleSize | MultiSize;
+export type SizeMapping = Array<GeneralSize>;
```
Comment (JonNorman) [inline static/src/javascripts/projects/commercial/types.js]: Nice!

**Review by JonNorman — APPROVED** [review]

**Comment by JonNorman** [comment]
Looks like the transpiler isn't a fan of the type definitions, but it otherwise looks awesome: I can use all these types in mine :)

**Comment by regiskuckaertz** [comment]
Yeah, I think they should be stripped by babel, but they are not 😒

**Comment by JonNorman** [comment]
It may be the same with types as with functions: the transpiler seems to complain quite a lot with simultaneous definition and export. Have you tried defining the types above and then exporting them?

**Pull request merged**